pkgs/by-name/fl/fluxcd/package.nix

```diff
@@ -9,7 +9,7 @@
 
 let
   version = "2.3.0";
-  sha256 = "sha256-ZQs1rWI31qDo/BgjrmiNnEdR2OL8bUHVz+j5VceEp2k=";
+  hash = "sha256-ZQs1rWI31qDo/BgjrmiNnEdR2OL8bUHVz+j5VceEp2k=";
```

Comment:
This change change broke `fluxcd` update script logic. And it was already using SRI.
It took me a while to understand what was happening, since `hash` and `sha256` did not jump eyes and behavior was a bit odd. Fix at: #345690